backend/functions/src/triggers/on-create-bet.ts

```diff
@@ -145,11 +144,6 @@ export const onCreateBet = functions
     await giveUniqueBettorAndLiquidityBonus(contract, eventId, bettor, bet)
 
     await Promise.all([
-      // They may be selling out of a position completely, so only add them if they're buying
```

Comment:
I think bettors will still receive a resolution notification bc they'll have a contractMetric associated with the contract, so that's good.

Reply:
The sooner you merge this the sooner we can stop sending traders emails for every unlinked binary market resolution!